net-voip/ring-daemon/ring-daemon-99999999.ebuild

```diff
@@ -32,220 +32,146 @@ LICENSE="GPL-3"
-	system-flac? ( >=media-libs/flac-1.3.0 )
-	system-gcrypt? ( >=dev-libs/libgcrypt-1.6.5 )
-	system-gmp? ( >=dev-libs/gmp-6.1.0 )
-	system-gnutls? ( >=net-libs/gnutls-3.4.14 )
-	system-gpg-error? ( >=dev-libs/libgpg-error-1.15 )
-	system-gsm? ( >=media-sound/gsm-1.0.13 )
-	system-iconv? ( virtual/libiconv )
-	system-jack? ( >=media-sound/jack-audio-connection-kit-0.121.3 )
-	system-jsoncpp? ( >=dev-libs/jsoncpp-1.7.2 )
-	system-msgpack? ( >=dev-libs/msgpack-2.1.0 )
-	system-nettle? ( >=dev-libs/nettle-3.1 )
-	system-ogg? ( >=media-libs/libogg-1.3.1 )
-	system-opendht? ( >=net-libs/opendht-1.3.0 )
-	system-opus? ( >=media-libs/opus-1.1.2 )
-	system-pcre? ( >=dev-libs/libpcre-8.40 )
+RDEPEND="system-gnutls? ( >=net-libs/gnutls-3.4.14 )
 	system-pjproject? ( >=net-libs/pjproject-2.5.5:2/9999 )
-	system-portaudio? ( >=media-libs/portaudio-19_pre20140130 )
-	system-restbed? ( >=net-libs/restbed-4.5 )
-	system-samplerate? ( >=media-libs/libsamplerate-0.1.8 )
-	system-sndfile? ( >=media-libs/libsndfile-1.0.25 )
-	system-speex? ( >=media-libs/speex-1.0.5 )
-	system-upnp? ( >=net-libs/libupnp-1.6.19:= )
-	system-uuid? ( sys-apps/util-linux )
-	system-vorbis? ( >=media-libs/libvorbis-1.3.4 )
-	system-vpx? ( >=media-libs/libvpx-1.6.0 )
-	system-x264? ( >=media-libs/x264-0.0.20140308 )
-	system-yaml-cpp? ( >=dev-cpp/yaml-cpp-0.5.3 )
-	system-zlib? ( >=sys-libs/zlib-1.2.8 )
-	pulseaudio? ( media-sound/pulseaudio )
-	dev-libs/dbus-c++
-	x11-libs/libva
-	"
-
-RDEPEND="${DEPEND}"
 
-src_configure() {
-	cd contrib
+	>=dev-cpp/yaml-cpp-0.5.3
 
-	# remove folders for other OSes
-	# android
-	rm -r src/natpmp
-	rm -r src/pthreads
-	rm -r src/speexdsp
-	rm -r src/libav
+	>=dev-libs/boost-1.61.0
+	>=dev-libs/crypto++-5.6.5
+	>=dev-libs/jsoncpp-1.7.2
 
-	if use system-asio; then
-		rm -r src/asio
-	fi
+	>=media-libs/libsamplerate-0.1.8
+	>=media-libs/libsndfile-1.0.25[-minimal]
 
-	if ! use system-boost; then
-		# boost is failing with a compilation error
-		# patch boost
-		sed -i.bak 's/\.\/b2/\.\/b2 --ignore-site-config /g' src/boost/rules.mak
-	fi
+	!libav? (	>=media-video/ffmpeg-3.1.3[encode,gsm?,iconv,libilbc?,opus?,speex?,v4l,vaapi?,vdpau?,vorbis?,vpx?,x264?,zlib] )
+	libav? ( >=media-video/libav-12:0=[encode,gsm?,opus?,speex?,v4l,vaapi?,vdpau?,vorbis?,vpx?,x264?,zlib] )
 
-	if use system-boost; then
-		rm -r src/boost
-	fi
+	libilbc? ( media-libs/libilbc )
 
-	if use system-cryptopp; then
-		rm -r src/cryptopp
-	fi
+	>=net-libs/opendht-1.3.0
+	>=sys-libs/zlib-1.2.8
+		x11-libs/libva
 
-	if use system-ffmpeg; then
-		rm -r src/ffmpeg
-	fi
+	alsa? ( media-libs/alsa-lib )
+	jack? ( virtual/jack )
+	portaudio? ( >=media-libs/portaudio-19_pre20140130 )
+	pulseaudio? ( media-sound/pulseaudio[alsa?,libsamplerate] )
 
-	if use system-flac; then
-		rm -r src/flac
-	fi
+	dbus? ( dev-libs/dbus-c++ )
+	ringns? ( >=net-libs/restbed-4.5 )
+	restbed? ( >=net-libs/restbed-4.5 )
+	sdes? ( >=dev-libs/libpcre-8.40 )
+	video? ( virtual/libudev )
```

Comment:
Sorry for the late comment. Why is the video use-flag causing a dependency on virtual/libudev. Shouldn't it be around ffmpeg and libav?